docs/en/Extensions.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: global
+group: Features
+title: Under Storage Extensions
+priority: 7
+---
+
+* Table of Contents
```

Comment:
suggestion: I feel there are two types of audience:
- ppl who want to simply use extension
- ppl who want to develop extensions. 
it requires two different levels of details. I suggest we could make one section for user and another for dev.